lib/window.js

```diff
@@ -1,295 +1,276 @@
 var angular = require('angular');
 var module = angular.module('ui.winwin');
+var getAreaFromEvent = require('./area_finder')
+var AREAS = require('./areas');
 
-windowController.$inject = ['$scope'];
+module.controller('WindowController', ['$scope', function windowController($scope) {
+  var ctrl = this;
+  var panes = ctrl.panes = $scope.panes = [];
 
-function windowController($scope, $element) {
-	var ctrl = this;
-	var panes = ctrl.panes = $scope.panes = [];
-	ctrl.select = function(selectedPane) {
-		angular.forEach(panes, function(pane) {
-			if(pane.active && pane !== selectedPane) {
-				pane.active = false;
-				pane.onDeselect();
-			}
-		});
-		selectedPane.active = true;
-		selectedPane.onSelect();
-	}
+  ctrl.select = function(selectedPane) {
+    angular.forEach(panes, function(pane) {
+      if (pane.active && pane !== selectedPane) {
+        pane.active = false;
+        pane.onDeselect();
+      }
+    });
+    selectedPane.active = true;
+    selectedPane.onSelect();
+  }
 
-	ctrl.addPane = function(pane) {
-		panes.push(pane);
-		if(panes.length > 0) {
-			ctrl.select(pane);
-		}
-	}
-	ctrl.removePane = function(pane) {
-		console.log("Remove Pane");
-		console.log(pane);
-		var index = panes.indexOf(pane);
-		panes.splice(index, 1);
-		if(panes.length > 0) {
-			ctrl.select(panes[0]);
-		} else if(panes.length === 0 && !ctrl.isDestroying){
-			ctrl.isDestroying = true;
-			$scope.destroyMe();
-		}
-	}
-	ctrl.createPane = function(caption, content) {
-		
-	}
-}
+  ctrl.addPane = function(pane) {
+    panes.push(pane);
+    if (panes.length > 0) ctrl.select(pane);
+  }
+
+  ctrl.removePane = function(pane) {
+    var index = panes.indexOf(pane);
+    panes.splice(index, 1);
+    if (panes.length > 0) {
+      ctrl.select(panes[0]);
+    } else if (panes.length === 0 && !ctrl.isDestroying) {
+      ctrl.isDestroying = true;
+      $scope.destroyMe();
+    }
+  }
 
-module.controller('WindowController', windowController);
+  ctrl.createPane = function(_caption, _content) {}
+}]);
 
 
 function _getPane(paneData, paneResolverService) {
-	paneId = paneData.id;
-	var newPane;
-	if(paneId) {
-		var sourcePane = $('#'+ paneId);
-		var sourceContentPane = $('#'+ paneId+"-content");
+  var paneId = paneData.id;
+  var newPane;
+  if (paneId) {
+    var sourcePane = $('#' + paneId);
```

Comment:
this is the first part where we need to change to support multiple browser windows. Because it pulls it from the local DOM.